README.md

```diff
@@ -1 +1,23 @@
 # react-cli
+
+This repo is an exploration for making the tooling for React that provides the best developer experience.
+
+
+## Getting started should be dead simple
+
+Something like `npm install -g react-cli` then `react init cool-project`.
+
+## Gradual integration
+
+The initial setup should be super slim, but then you can add more to it. When you start to want to test your app, you can do `react add test`, if you want to use flow then `react add flow`, wanna add crash reporting then `react add sentry`...
+
+Once you added a module (for example `react add test`) to your setup, it is registered as an executable on `react`. So you can do `react test` or `react test --add MyModule.js`...
```

Comment:
Not all modules would need to be registered as an executable, for example `redux` probably wouldn't need its own executable.

It might make more sense to allow the module to define its executables (again similiar to how `npm` does it :smile: )